.github/workflows/ci.yaml

```diff
@@ -60,7 +60,7 @@ jobs:
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
       - name: Install Helm-unittest
-        run: helm plugin install https://github.com/quintush/helm-unittest
+        run: helm plugin install https://github.com/helm-unittest/helm-unittest
```

Comment:
I think this is part of the current failures...not sure why this would affect `hull` CI errors though. @jbiers